src/main/java/pl/wrapper/parking/facade/ParkingService.java

```diff
@@ -1,3 +1,17 @@
 package pl.wrapper.parking.facade;
 
-public interface ParkingService {}
+import pl.wrapper.parking.infrastructure.error.Result;
+import pl.wrapper.parking.pwrResponseHandler.dto.ParkingResponse;
+
+import java.util.List;
+
+public interface ParkingService {
+
+    Result<ParkingResponse> getByName(String name, Boolean opened);
+
+    Result<ParkingResponse> getById(Integer id, Boolean opened);
+
+    Result<ParkingResponse> getBySymbol(String symbol, Boolean opened);
+
+    Result<List<ParkingResponse>> getByParams(String symbol, Integer id, String name, Boolean opened);
```

Comment:
Return just the List. No need to wrap in Result as empty list is also a valid response. Add the @Nullable annotation to the arguments